nuget/Plugin.nuspec

```diff
@@ -62,6 +62,10 @@
      <file src="src\Media.Plugin.Abstractions\bin\Release\Plugin.Media.Abstractions.dll" target="lib\wpa81\Plugin.Media.Abstractions.dll" />
      <file src="src\Media.Plugin.Abstractions\bin\Release\Plugin.Media.Abstractions.xml" target="lib\wpa81\Plugin.Media.Abstractions.xml" />
      <file src="src\Media.Plugin.Abstractions\bin\Release\Plugin.Media.Abstractions.pdb" target="lib\wpa81\Plugin.Media.Abstractions.pdb" />
+     <file src="src\Media.Plugin.WindowsPhone81\bin\Release\CameraCaptureUI.xaml.cs" target="lib\wpa81\CameraCaptureUI.xaml.cs" />
```

Comment:
Does beta 5 still have the issues? Do we just want the files to be added to your app when you install the nuget?